doc: test commit (#6052)
nrfconnect/sdk-nrf: samples/zigbee/shell/README.rst

Button 4:
    Toggles the identification mode on the device.

> All other interactions with the application can be handled using **serial** communication.

Reviewer: ```suggestion
You can use **serial** communication for all other interactions with the application.
```
This is a dummy review suggestion.
